pkg/registry/core/adapters/nse_next_server.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adapters
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/networkservicemesh/api/pkg/api/registry"
+	"google.golang.org/grpc"
+)
+
+type nextNSEServer struct {
```

Comment:
`next` could confuse the reader, because we already have pkg `next`. Please consider renaming this one and all similar places.

Reply:
It is a wrapper around `next.NetworkServiceEndpointRegistryServer()`, not sure how to name it instead.

Reply:
@xzfc What about `callNextServer` or `adaptedServer`?

Reply:
OK, `callNextNSEServer`.